osu.Game/Screens/Menu/IntroScreen.cs

```diff
@@ -28,11 +28,18 @@ public abstract class IntroScreen : StartupScreen
 
         private Bindable<bool> menuVoice;
 
+        private LeasedBindable<WorkingBeatmap> beatmap;
+
+        public new Bindable<WorkingBeatmap> Beatmap { get => beatmap; }
+
         protected override BackgroundScreen CreateBackground() => new BackgroundScreenBlack();
 
         [BackgroundDependencyLoader]
         private void load(OsuConfigManager config, BeatmapManager beatmaps, Framework.Game game)
         {
+            //we take a lease on the beatmap bindable to prevent music playback from changing / pausing music during intros, as it is causing weird interactions with certains intros 
```

Comment:
```suggestion
            // we take a lease on the beatmap bindable to prevent music playback from changing / pausing during intros, as it is causing weird interactions with certains intros.
```